lincheck/src/main/java/org/jetbrains/kotlinx/lincheck/Reporter.kt

```diff
@@ -85,3 +81,34 @@ private fun <T> printInColumns(groupedObjects: List<List<T>>): String {
             .map { rowCells -> rowCells.joinToString(separator = " | ", prefix = "| ", postfix = " |") }
             .joinToString(separator = "\n")
 }
+
+private class ActorWithResult(val actorRepresentation: String, val spaces: Int, val resultRepresentation: String) {
+    override fun toString(): String = actorRepresentation + ":" + " ".repeat(spaces) + resultRepresentation
+}
+
+private fun uniteActorsAndResults(actors: List<Actor>, results: List<Result>): List<ActorWithResult> {
+    require(actors.size == results.size) {
+        "Different number of actors and matching results found (${actors.size} != ${results.size})"
+    }
+
+    val actorRepresentations = actors.map { it.toString() }
+    val resultRepresentations = results.map { it.toString() }
+
+    val maxActorLength = actorRepresentations.map { it.length }.max()
```

Comment:
put "!!" here

Reply:
Then this code won't work if we have zero actors (currently it works).